### PR TITLE
Create endpoint for changing in user data without frontend

### PR DESCRIPTION
**Pull request by vasylneshchadimov**

### CODE REVIEW

File: backend/routes/index.ts [hunk 1]
```diff
@@ -64,5 +64,10 @@ router.get(
   }),
   AuthController.signInFacebook
 );
+router.put('/user',
+upload.single('image'),
+UserController.changeUserData
+);
+
```
Comment (6mahdapihka9) [inline backend/routes/index.ts]: make it patch method and mb path '/profile' or '/user-data'

File: backend/routes/index.ts [hunk 2]
```diff
@@ -64,5 +64,10 @@ router.get(
   }),
   AuthController.signInFacebook
 );
+router. patch('/profile',
```
Comment (Yojuks) [inline backend/routes/index.ts]: remove gap

**Review by Yojuks — CHANGES_REQUESTED** [review]

File: backend/routes/index.ts (hunk 2 above)
Comment (vasylneshchadimov) [reply backend/routes/index.ts]: done

**Review by romawht22 — APPROVED** [review]

**Review by Yojuks — APPROVED** [review]

File: backend/controllers/UserController.ts [hunk 3]
```diff
@@ -27,7 +27,37 @@ const UserController = {
...
+          $set: {
+            ...newUserData
+          }
+        },
+        {
+          new: true
+        }
+    );
+    if (!changeData) {
+      return res.status(400).json({ error: "User doesn't exist" });
```
Comment (steadyGuy) [inline backend/controllers/UserController.ts:53]: add to translation

**Review by steadyGuy — APPROVED** [review]

**Review by steadyGuy — APPROVED** [review]

**Pull request merged**